flask_api/api_files_db.py

```diff
@@ -0,0 +1,36 @@
+import json
+import os
+
+
+class Filedb:
+    def __init__(self):
+        # get dir to file folder (create if dont exist)
+        root_dir = os.path.dirname(os.getcwd())
+        self.SAVE_DIR = os.path.join(root_dir, 'files' )
+        os.makedirs(self.SAVE_DIR, exist_ok=True)
+
+
+    def save(self, obj, name):
```

Comment:
If you dont choose to use documentation, pick better variable names and function names and add type annotations. you code is not clear enough, if you want you code to be self explanatory you put more effort into picking good variable names